service/r2/routes.go

```diff
@@ -212,16 +214,16 @@ func createRollupRule(s *service, r *http.Request) (data interface{}, err error)
 		return nil, err
 	}
 
-	rr, err := s.store.CreateRollupRule(namespaceID, rrj.rollupRuleView(), uOpts)
+	_, err = s.store.CreateRollupRule(namespaceID, rrj.ToRollupRuleView(), uOpts)
 	if err != nil {
 		return nil, err
 	}
-	return newRollupRuleJSON(rr), nil
+	return rrj, nil
```

Comment:
Hm why the change here?

Reply:
I was thinking that if we already allocated the exact same object when deserializing the body of the request we dont need to do it again with the view that comes back from the store package. It might be a premature optimization, I can get rid of it.